examples/all-clusters-app/esp32/main/CHIPDeviceManager.cpp

```diff
@@ -74,6 +77,13 @@ CHIP_ERROR CHIPDeviceManager::Init(CHIPDeviceManagerCallbacks * cb)
         ConnectivityMgr().SetBLEAdvertisingEnabled(false);
         ConnectivityMgr().SetWiFiAPMode(ConnectivityManager::kWiFiAPMode_Enabled);
     }
+    else if (flags.Has(RendezvousInformationFlag::kOnNetwork))
+    {
+        ConnectivityMgr().SetBLEAdvertisingEnabled(false);
+        ConnectivityMgr().SetWiFiAPMode(ConnectivityManager::kWiFiAPMode_Disabled);
+        DeviceNetworkProvisioningDelegateImpl deviceDelegate;
+        deviceDelegate.ProvisionWiFi(CONFIG_EXAMPLE_WIFI_SSID, CONFIG_EXAMPLE_WIFI_PASSWORD);
```

Comment:
@wqx6 If these configs are used for ssid and passphrase then what is the purpose of: https://github.com/project-chip/connectedhomeip/blob/master/config/esp32/components/chip/Kconfig#L326-L344

Can anyone of these be removed, so that it is clear?